pydicom/sr/codedict.py

```diff
@@ -250,10 +338,10 @@ def dir(self, *filters):
         """
         return _filtered(chain.from_iterable(self._dict.values()), filters)
 
-    def schemes(self):
+    def schemes(self) -> KeysView[str]:
         return self._dict.keys()
 
-    def trait_names(self):
+    def trait_names(self) -> List[str]:
         """Returns a list of valid names for auto-completion code.
 
         Used in IPython, so that data element names can be found and offered
```

Comment:
Maybe @pieper can have another look at the refactoring in the module - it all looks fine to me, but I'm not really involved in the SR stuff.